crates/cheatcodes/src/inspector.rs

```diff
@@ -1412,12 +1481,14 @@ fn disallowed_mem_write(
 
 /// Expands memory, stores a revert string, and sets the return range to the revert
 /// string's location in memory.
-fn mstore_revert_string(interpreter: &mut Interpreter<'_>, bytes: &[u8]) {
+fn mstore_revert_string(interpreter: &mut Interpreter, bytes: &[u8]) {
     let starting_offset = interpreter.shared_memory.len();
     interpreter.shared_memory.resize(starting_offset + bytes.len());
     interpreter.shared_memory.set_data(starting_offset, 0, bytes.len(), bytes);
-    interpreter.return_offset = starting_offset;
-    interpreter.return_len = interpreter.shared_memory.len() - starting_offset
+
+    // TODO(mattsse): what happened to this??
+    // interpreter.return_offset = starting_offset;
+    // interpreter.return_len = interpreter.shared_memory.len() - starting_offset
```

Comment:
This is now in `CallOutcome`, I think we want to pass `&mut CallOutcome` to this function

Reply:
Actually we don't have that, this is during `step`